core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,30 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
```

Comment:
Also -- if it didn't show up on the admin page, how did you do manual testing for this job?